Doc/library/array.rst

```diff
@@ -48,13 +48,10 @@ defined:
 Notes:
 
 (1)
-   The ``'u'`` type code corresponds to Python's obsolete unicode character
+   The ``'u'`` type code had corresponded to Python's obsolete unicode character
    (:c:type:`Py_UNICODE` which is :c:type:`wchar_t`). Depending on the
    platform, it can be 16 bits or 32 bits.
```

Comment:
You can copy/paste what you wrote in What's New in Python 3.9 here. The current (1) not doesn't mention in which version it changed. Maybe added a ".. versionchanged:: 3.9" markup.